api/settings.py

```diff
@@ -114,8 +113,8 @@
 # REST FRAMEWORK settings
 
 REST_FRAMEWORK = {
-    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.PageNumberPagination',
-    'PAGE_SIZE': 10,
+    # 'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.PageNumberPagination',
```

Comment:
:thinking: commented?

Reply:
I was thinking of paginating the API list view but then I changed my mind